CODEOWNERS

```diff
@@ -6,14 +6,15 @@
 #
 # Learn about membership in OpenTelemetry community:
 #  https://github.com/open-telemetry/community/blob/main/community-membership.md
-# 
 #
-# Learn about CODEOWNERS file format: 
+#
+# Learn about CODEOWNERS file format:
 #  https://help.github.com/en/articles/about-code-owners
 #
 
 @open-telemetry/erlang-approvers
 
+/instrumentation/opentelemetry_broadway         @bryannaegele @tsloughter @whatyouhide @tomtaylor
```

Comment:
If we want this, you folks gotta add us to a team that has access to this repo, I think.